Automattic/wp-calypso
# Domains: Update the support docs link for Designated Agent

#### Changes proposed in this Pull Request
We're removing the [dedicated page for Designated Agent](https://wordpress.com/support/designated-agent/) and keeping just the [consolidated one](https://wordpress.com/support/update-contact-information/#designated-agent). We'll be adding a redirect from the old page to the new one, but it's still best to update the link to avoid unnecessary network requests.

#### Testing instructions

For a domain that supports Designated Agent (gTLDs at Key-Systems) go to Domains > example.com > Update your contact information > Edit contact info and make sure the link at the bottom points to the correct place:


<img width="704" alt="Screenshot 2020-07-16 at 13 31 15" src="https://user-images.githubusercontent.com/3392497/87677006-cb539480-c768-11ea-9350-371fc467c7d8.png">
